addons/tmf_loadouts/loadouts/uk_rm_1982.hpp

```diff
@@ -0,0 +1,512 @@
+// If you want to base a loadout on an existing one, this repository contains them all:
+// https://github.com/ARCOMM/arc_misc/tree/master/addons/tmf_loadouts
+
+//Dependencies:
+//		* CUP Units
+//		* CUP Weapons
+//		* Community Factions Project
+//		* FIR AWS(AirWeaponSystem)
+//		* MNP Uniforms
+//		* HLC FAL Pack
+//		* HLC MP5 Pack
+//		* ACE
+
+//		Note: Just swap the berets and you have a para regiment instead.
+//		Addtl. Note: Helmets are the only armor on this kit.
+
+//	Created by: AChesheireCat
+
+class baseMan {// Weaponless baseclass
+	displayName = "Unarmed";
+	// All randomized.
+	uniform[] = {
+		LIST_2("CUP_U_B_BDUv2_dirty_DPM"),
+		LIST_2("CUP_U_B_BDUv2_DPM"),
+		"CUP_U_B_BDUv2_dirty_DPM_OD",
+		"CUP_U_B_BDUv2_DPM_OD"
+	};
+	vest[] = {"CFP_Alice_Vest2"};
```

Comment:
IMO the CUP Alice vest looks more like the ones the brits used

Reply:
Unfortunately the CUP ALICE gear sits a couple inches off of the BDU used. I could change it if we really wanted to though.